source/connection-troubleshooting.txt

```diff
@@ -74,27 +75,27 @@ instance, it generates this error message:
 .. code-block:: none
    :copyable: false
 
-   MongoServerSelectionError: connect ECONNREFUSED ::<IP address>:<port>
+   MongoServerSelectionError: connect ECONNREFUSED <IPv6 address>:<port>
 
 The following sections describe actions you can take to potentially resolve the
 issue.
 
-Start MongoDB in IPv6 Mode
---------------------------
+Ensure MongoDB and Your Client Use the Same Protocol
+----------------------------------------------------
 
 In Node.js v17 and later, the DNS resolver uses ``IPv6`` by default when both
-the client and host support both. You can configure your MongoDB deployment
-to use ``IPv6`` mode when starting with ``mongod`` or ``mongos``. For more
-information about how to specify ``IPv6`` mode, see
+the client and host support both. If MongoDB uses one mode (such as IPv4) and your
+client uses a different mode (such as IPv6), this will result in the error message
+above.
```

Comment:
Suggestion:
The [style guide advises against using parentheses](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#parentheses) other than for a select few reasons.


```suggestion
the client and host support both. For example, if MongoDB uses IPv4 and your client uses a different mode, such as IPv6, the driver returns the prior error message.
```